build/repo.targets

```diff
@@ -0,0 +1,71 @@
+<Project>
+  <ItemGroup>
+    <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1" />
+    <Scenarios Include="-n Plaintext -o KestrelHttpServer@feature/dev-si --kestrelThreadCount 2" />
```

Comment:
Another idea to make this easier on command line is to give these friendly names like this:
```xml
<Scenario Include="Plaintext">
  <Args>-n Plaintext -o KestrelHttpServer@feature/dev-si --kestrelThreadCount 2</Args>
</Scenario>
```